dll/win32/kernel32/client/sysinfo.c

```diff
@@ -380,13 +380,36 @@ GetNumaProcessorNode(IN UCHAR Processor,
     return TRUE;
 }
 
-/*
- * @implemented
- */
+/**
+ * 
+ * @name GetNumaAvailableMemoryNode
+ * @implemented NT 5.2
+ *
+ *     Retrieves the amount of memory available in the specified node.
+ *
+ * @param Node
+ *        The number of the node.
+ *
+ * @param AvailableBytes
+ *        The amount of available memory for the node, in bytes.
+ *
+ * @return If the function succeeds, the return value is non zero.
+ *         If the function fails, the return value is zero. To get
+ *         extended error information, call GetLastError.
+ *
+ * @remarks The GetNumaAvailableMemoryNode function returns the amount of memory consumed
```

Comment:
These comments use the exact same wording as MSDN. This is not acceptable, those texts are under copyright. You'll have to describe the function in your own words.